Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -174,6 +160,51 @@ public final class MapboxMap {
     }
 }
 
+extension MapboxMap: MapTransformDelegate {
```

Comment:
on Android these are not extension functions, but the functions directly inside MapboxMap, but I think this is just a implementation detail for iOS.